File: lab3/increDecre.c
#include <stdio.h>
 
main()
{
 int i=1;
 i++; 
 printf("i is %d\n", i);

 i=1;
 ++i;
 printf("i is %d\n------------\n", i);

 i=1;
 printf("i is %d\n", i++);
 printf("i is %d\n-----\n",i);
 
 i=1;
 printf("i is %d\n", ++i);
 printf("i is %d\n-------------\n",i);
 
 i=1;
 int y = i++;
 printf("i is %d, y is %d\n", i, y);

 i=1;
 y = ++i;
 printf("i is %d, y is %d\n-------------\n", i, y);

 // use ++ -- to make code succlent
 int arr[10];
 int k=0; int v=100;
 while (k<10)
   arr[k++] = v--;   // instead of arr[k]=v; k++; v--;

 int index;
 for(index=0; index<10; index++)
    printf("arr[%d]: %d\n", index,arr[index]);
}

File: lab3/IncreDecre.java
public class IncreDecre{ 
 public static void main(String[] args)
{
 int i=1;
 i++; 
 System.out.printf("i is %d\n", i);

 i=1;
 ++i;
 System.out.printf("i is %d\n-----------\n", i);

 i=1;
 System.out.printf("i is %d\n", i++);
 System.out.printf("i is %d\n------\n", i);
 i=1;
 System.out.printf("i is %d\n", ++i);
 System.out.printf("i is %d\n-----------\n",i);

 i=1;
 int y = i++;
 System.out.printf("i is %d, y is %d\n", i, y);

 i=1;
 y = ++i;
 System.out.printf("i is %d, y is %d\n-----------\n", i, y);

 int arr[] = new int[10];
 int k=0; int v=100;
 while (k<10)
   arr[k++] = v--;

 int index=0;
 for(index=0; index<10; index++)
    System.out.printf("arr[%d]: %d\n", index,arr[index]);
 }

}
